samples/MvvmSample.Core/ViewModels/MessengerPageViewModel.cs

```diff
@@ -3,20 +3,38 @@
 // See the LICENSE file in the project root for more information.
 
 using Microsoft.Toolkit.Mvvm.ComponentModel;
+using Microsoft.Toolkit.Mvvm.Input;
 using Microsoft.Toolkit.Mvvm.Messaging;
 using Microsoft.Toolkit.Mvvm.Messaging.Messages;
+using System.Windows.Input;
 
-namespace MvvmSampleUwp.ViewModels
+namespace MvvmSample.Core.ViewModels
 {
     public class MessengerPageViewModel : SamplePageViewModel
     {
+		public MessengerPageViewModel()
+		{
```

Comment:
Small indentation issue 🙂
Also make sure indentations are using 4 spaces and not TAB characters!

```suggestion
        public MessengerPageViewModel()
        {
```

Reply:
> make sure indentations are using 4 spaces and not TAB characters!

Just when I thought we could be friends

Reply:
We can still be! 😭